standards/mobile_app_standards.md

```diff
@@ -0,0 +1,135 @@
+
+Caveats
+-------
+
+The huge increase in the use of mobile devices means that mobile application development is now of critical interest to the DEFRA family, even more so given the amount of field-work being carried out by workers within the DEFRA family.
+
+This is a fast moving area involving multiple platforms and diverse technology stacks: these standards are therefore early and provisional. As such, these standards often require more background explanation and justification than those for other, more mature or more homogenous, technology stacks. Inevitably, therefore, these standards, for the time being, are likely to be more discursive than those for other areas, and, unavoidably, the line between standard and guidance can often, as yet, not be so sharply drawn.
+
+This document should therefore be read in conjunction with the Mobile Application Guidance.
+
+Which Mobile Platforms Must Be Supported
+----------------------------------------
+
+Mobile applications must run on mobile devices. Currently DEFRA has adopted Apple technology as the standard platform for mobile devices. **All mobile applications intended for internal use must therefore run on Apple devices. All mobile applications intended for use beyond the DEFRA family, by any form of "extended enterprise" or the general public, must run on at least Apple and Android devices.**
+
+Working Offline
+---------------
+
+Users of mobile devices, and especially DEFRA family field workers who will often be working in remote, rural locations, must be able to continue to use any mobile applications even when lack of signal prohibits any form of internet access. In fact, **continuing to be readily usable without an internet connection, including the ability to work with data locally on the device, must be considered a defining characteristic of a truly mobile app.**
+
+Extra Security Considerations
+-----------------------------
+
+Application and data security is always essential, and mobile apps can be especially exposed to security risks and challenges. All mobile devices suffer from poor physical security in terms of being easily lost or stolen, and all have to operate over potentially insecure public networks. Furthermore, when an app is intended to run on a public device, then the security of the device is an unknown, and, by default, we should assume the device is inherently insecure. This means that the points below **must be considered and addressed very early in any project to develop such a mobile app**, as they could even influence the practical viability of the app or the scope of the requirements it can securely deliver.
```

Comment:
This doesn't tell me what the standard is that I'm going to be measured against.

I'd like to see something that says "Define/design the complete security profile of your app before you build it".

Reply:
As agreed elsewhere, I'll add that as a standard.

What that para is trying to say though is that the security challenges related to mobile devices are sufficient to mean that delivering functionality that requires very high security via a mobile solution might not be appropriate in the first place: therefore security should be though about before agreeing scope/MVP.